flake.nix

```diff
@@ -23,13 +23,16 @@
             inherit pkgs falsisign-src;
           };
         in
-        {
-          apps = {
+		{
+		  apps.x86_64-linux= {
```

Comment:
This should probably be `apps.${system} = { ... } `, otherwise this will only work on this specific platform.

Reply:
Fixed